packages/api/app/Models/Technology.js

```diff
@@ -88,6 +97,27 @@ class Technology extends Model {
 		return query.where({ slug: technology });
 	}
 
+	/**
+	 * Gets a technology by its id or slug
+	 *
+	 * @param {string|number} technology Technology id or slug.
+	 * @returns {Technology}
+	 */
+	static async getTechnology(technology) {
+		if (!Number.isNaN(parseInt(technology, 10))) {
+			return Technology.findOrFail(technology);
+		}
+
+		const technologyInst = await this.query()
+			.where({ slug: technology.toUpperCase() })
+			.first();
```

Comment:
tem algum motivo especial aqui para ter usado o `this.query` ao inves da `Technology` ?

Reply:
Se você achar melhor eu posso usar essa sintaxe:
`const technologyInst = await Technology.findBy('slug', technology.toUpperCase());`
Dá no mesmo.

Reply:
Não tem problema @alexandreadames, era só para tentar perceber se tinha algum motivo especial em usar isso 👍  pode deixar assim